packages/commonjs-shim/README.md

```diff
@@ -0,0 +1,49 @@
+[npm]: https://img.shields.io/npm/v/@rollup/plugin-commonjs-shim
+[npm-url]: https://www.npmjs.com/package/@rollup/plugin-commonjs-shim
+[size]: https://packagephobia.now.sh/badge?p=@rollup/plugin-commonjs-shim
+[size-url]: https://packagephobia.now.sh/result?p=@rollup/plugin-commonjs-shim
+
+[![npm][npm]][npm-url]
+[![size][size]][size-url]
+[![libera manifesto](https://img.shields.io/badge/libera-manifesto-lightgrey.svg)](https://liberamanifesto.com)
+
+# @rollup/plugin-commonjs-shim
+
+🍣 A Rollup plugin to replace CJS syntax like `__filename`, `__dirname`, ... for esm output bundles.
```

Comment:
```suggestion
🍣 A Rollup plugin to replace CJS global variables within esm output bundles.
```

Let's list all of them that are shimmed directly below this the above;

```
The list of global variables that are shimmed are:
- ...
```

Reply:
done.